src/types/action.ts

```diff
@@ -38,3 +39,9 @@ export type SubmitLifecycleIdForm = {
   nextSteps?: string;
   feedback?: string;
 };
+
+export type RejectIntakeForm = {
+  feedback?: string;
```

Comment:
I think these should all be required since we have form validations for them.

Reply:
Makes sense. I can change that.